### PR TITLE
Add regression test udq_actionx/UDQ_M1

### PR DESCRIPTION
**Pull request by joakim-hove**
Add regression test for UDQ functions `UADD / UMUL / UMIN / UMAX`

### CODE REVIEW

**Comment by joakim-hove** [comment]
When this is confirmed from Equinor I will merge along with updated data

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by joakim-hove** [comment]
jenkins build this update_data please

**Comment by joakim-hove** [comment]
OK - the model runs in flow and the `update_reference_data.sh` script has been updated; I fold

File: tests/update_reference_data.sh [hunk 1]
```diff
@@ -126,6 +126,7 @@ tests[polymer_injectivity]="flow polymer_injectivity 2D_POLYMER_INJECTIVITY"
 tests[nnc]="flow editnnc NNC_AND_EDITNNC nnc"
 tests[udq_wconprod]="flow udq_actionx UDQ_WCONPROD udq_wconprod"
 tests[udq_actionx]="flow udq_actionx UDQ_ACTIONX udq_actionx"
+tests[udq_M1]="flow udq_actionx UDQ_M1 udq_M1"
```
Comment (akva2) [inline tests/update_reference_data.sh:129]: "flow udq_actionx UDQ_M1 udq_uadd"

**Comment by joakim-hove** [comment]
jenkins build this update_data please

File: tests/update_reference_data.sh (hunk 1 above)
Comment (joakim-hove) [reply tests/update_reference_data.sh]: OK thank you; I am trying that now. But I do not understand what was wrong with my initial attempt.

As time has gone by the common pattern is that the directories have multiple test files in them, maybe the way to add testdata should treat/describe that as the normal (only ??) way to do it?

Comment (akva2) [reply tests/update_reference_data.sh]: the last parameter must match the cmake test name

**Comment by akva2** [comment]
jenkins build this opm-tests=413 please

**Pull request merged**